cookbook-react/recipes/inputs/barcelona/index.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import { render, queryByAttribute }  from '@testing-library/react';
+
+import Input from '.';
+
+const getById = queryByAttribute.bind(null, 'data-testid');
+
+describe('Barcelona Input', () => {
+  const staticProps = { label: 'My Label', placeholder: 'Some Placeholder', name: "email"  }
```

Comment:
```suggestion
  const staticProps = { label: 'My Label', placeholder: 'Some Placeholder', name: 'email'  }
```
Are test files being linted?